docs/java-rest/high-level/usage.asciidoc

```diff
@@ -9,15 +9,25 @@ getting the artifact to using it in an application.
 The Java High Level REST Client requires Java 1.8 and depends on the Elasticsearch
 core project. The client version is the same as the Elasticsearch version that the
 client was developed for. It accepts the same request arguments as the `TransportClient`
-and returns the same response objects.
-
-The High Level Client is backwards compatible but can only communicate with Elasticsearch
-version 5.5 and onwards. The High Level Client is forward compatible as well, meaning that
-it supports communicating with a later version of Elasticsearch than the one it was developed
-for. It is recommended to upgrade the High Level Client when upgrading the Elasticsearch
-cluster to a new major version, as REST API breaking changes may cause unexpected results,
-and newly added APIs will only be supported by the newer version of the client. The client
-should be updated last, once all of the nodes in the cluster have been upgraded.
+and returns the same response objects. See the <<java-rest-high-level-migration>>
+if you are needing to migrate an application from `TransportClient` to the
```

Comment:
nit: I'm not a native speaker, so maybe @nik9000 can judge this better, but "you are needing to" sounds odd to me. For me "need to", "want to" or "are migrating" sounds better